testing/ef-tests/src/macros/fork_choice.rs

```diff
@@ -0,0 +1,256 @@
+                };
+                use ream_fork_choice::{
+                    handlers::{on_attestation, on_attester_slashing, on_block, on_tick},
+                    store::{get_forkchoice_store, Store},
+                };
+                use rstest::rstest;
+                use serde::Deserialize;
+                pub const MAX_BLOB_COMMITMENTS_PER_BLOCK: u64 = 4096;
+                use ssz_derive::{Decode, Encode};
+                use ssz_types::{
+                    typenum::{self, U1099511627776, U4096}, FixedVector, VariableList
+                };
+                use tree_hash::TreeHash;
+
+                use super::*;
+                use $crate::utils;
+
+                #[derive(Debug, Deserialize)]
+                pub struct Tick {
+                    pub tick: u64,
+                    #[serde(default)]
+                    pub valid: Option<bool>,
+                }
+
+                #[derive(Debug, Deserialize)]
+                pub struct ShouldOverrideForkchoiceUpdate {
+                    pub validator_is_connected: bool,
+                    pub result: bool,
+                }
+
+                #[derive(Debug, Deserialize)]
+                pub struct Head {
+                    pub slot: u64,
+                    pub root: B256,
+                }
+
+                #[derive(Debug, Deserialize)]
+                pub struct Checks {
+                    #[serde(default)]
+                    pub head: Option<Head>,
+                    #[serde(default)]
+                    pub time: Option<u64>,
+                    #[serde(default)]
+                    pub justified_checkpoint: Option<Checkpoint>,
+                    #[serde(default)]
+                    pub finalized_checkpoint: Option<Checkpoint>,
+                    #[serde(default)]
+                    pub proposer_boost_root: Option<B256>,
+                    #[serde(default)]
+                    pub get_proposer_head: Option<B256>,
+                    #[serde(default)]
+                    pub should_override_forkchoice_update: Option<ShouldOverrideForkchoiceUpdate>,
+                }
+
+                #[derive(Debug, Deserialize)]
+                pub struct Block {
+                    pub block: String,
+                    pub blobs: Option<String>,
+                    pub proofs: Option<Vec<String>>,
+                    pub valid: Option<bool>,
+                }
+
+                #[derive(Debug, Deserialize)]
+                pub struct AttestationStep {
+                    pub attestation: String,
+                    #[serde(default)]
+                    pub valid: Option<bool>,
+                }
+
+                #[derive(Debug, Deserialize)]
+                pub struct AttesterSlashingStep {
+                    pub attester_slashing: String,
+                    #[serde(default)]
+                    pub valid: Option<bool>,
```

Comment:
```suggestion
                #[derive(Debug, Deserialize)]
                pub struct AttesterSlashingStep {
                    pub attester_slashing: String,
                    pub valid: Option<bool>,
```

do you need default?